packages/paste-website/src/pages/components/sidebar-navigation/index.mdx

```diff
@@ -0,0 +1,849 @@
+| ------------ | ----------------- | ----------------------------------------------------------------------------------------- | ----------------------------------------- |
+| `element?`   | `string`          | Overrides the default element name to apply unique styles with the Customization Provider | `'SIDEBAR_NAVIGATION_DISCLOSURE_HEADING'` |
+| `children?`  | `React.ReactNode` |                                                                                           |                                           |
+| `disabled?`  | `boolean`         | Set to disable the ability to expand the Navigation section                               |                                           |
+| `focusable?` | `boolean`         | Set to control the focusability of the Navigation Heading                                 |                                           |
+| `selected?`  | `boolean`         | Set to make the Heading part of the selected page hierarchy                               |                                           |
+| `icon?`      | `React.ReactNode` | Icon to be displayed within the Heading                                                   |                                           |
+
+##### SidebarNavigationDisclosureContent
+
+| Prop        | Type              | Description                                                                               | Default                                   |
+| ----------- | ----------------- | ----------------------------------------------------------------------------------------- | ----------------------------------------- |
+| `element?`  | `string`          | Overrides the default element name to apply unique styles with the Customization Provider | `'SIDEBAR_NAVIGATION_DISCLOSURE_CONTENT'` |
+| `children?` | `React.ReactNode` |                                                                                           |                                           |
+
+##### SidebarNavigationSeparator
+
+| Prop       | Type     | Description                                                                               | Default                          |
+| ---------- | -------- | ----------------------------------------------------------------------------------------- | -------------------------------- |
+| `element?` | `string` | Overrides the default element name to apply unique styles with the Customization Provider | `'SIDEBAR_NAVIGATION_SEPARATOR'` |
+
+##### useSidebarNavigationDisclosureState
+
+Pass these as part of an object to the hook.
+
+| Prop       | Type      | Description                                                   | Default |
+| ---------- | --------- | ------------------------------------------------------------- | ------- |
+| `baseId?`  | `string`  | Set to create a base ID that all dynamic IDs will be based on |         |
+| `visible?` | `boolean` | Set the initial visibilty of the Disclosure                   |         |
```

Comment:
```suggestion
| `visible?` | `boolean` | Set the initial visibility of the Disclosure                   |         |
```

Reply:
caught one more typo, but I'll approve.

Reply:
Not much point in approving. When I accept it'll blow it away

Reply:
it makes me feel better